frontend/hooks/data-fetching/course.ts

```diff
@@ -1,4 +1,11 @@
+import { useEffect } from "react";
 import useSWR, { mutate as globalMutate } from "swr";
+import {
+    useRealtimeResourceList,
+    useRealtimeResource,
+} from "@pennlabs/rest-live-hooks";
+// TODO: REMOVE THIS AS SOON AS WE REFACTOR
```

Comment:
Just wanted to point this out, I think this is the best solution for now - I don't want to spend time refactoring out everything and delaying websockets, but their types don't coerce well so I have to import this version